deployment-guides/AWS-EKS-EFS/README.md

```diff
@@ -0,0 +1,60 @@
+# Initialise the Hinxton Galaxy portal on AWS
+
+Scripts in this directory allow initalisation of Galaxy (of the Hinxton single-cell flavour) on AWS using Kubernetes under EKS using Amazon's elastic file system (EFS).
+
+## Configure
+
+The following assumes you have the AWS client installed and configured properly with all necessary authenticaion.
+
+### Clone respository
+
+```
+cd <checkout dir>
+git clone git@github.com:ebi-gene-expression-group/galaxy-kubernetes.git
+```
+
+### Environment
+
+Edit `<checkout_dir>/deployment-guides/AWS-EKS-EFS/destroy-all.sh`. In particular you'll need to modify DEVELOPER and AWS_REGION:
+
+```
+export DEVELOPER=<your amazon account ID>
+export AWS_REGION=<an Amazon region, e.g. us-west-2>
+```
```

Comment:
Why are the above settings (authentication) on the destroy-all.sh file? shouldn't they be in a general config file outside of the version controlled ones - you could have a .sample for instance.

Reply:
I guess that by developer, you mean an AWS key, right? Could you also comment on the authorisation level (policies) that that user needs to have access to execute all of these? It must be the PowerUser role I guess.

Reply:
Ahh, I see that they are in an env file, is just the docs here that need polishing. You probably want to mention here as well that the aws-cli needs to be authenticated with an aws key (with the adequate permissions).

Reply:
So, 'developer' is the key you need to pass to 'aws efs create-file-system', hence the variable name- have corrected the README file